HealthGPT/HealthGPTStandard.swift

```diff
@@ -8,9 +8,10 @@
 
 import Spezi
 import SpeziHealthKit
+import SwiftUI
 
 
-actor HealthGPTStandard: Standard, ObservableObject, ObservableObjectProvider, HealthKitConstraint {
+actor HealthGPTStandard: Standard, ObservableObject, HealthKitConstraint {
```

Comment:
Does this really need to conform to `ObservableObject, HealthKitConstraint`?

Reply:
Can remove `ObservableObject`, but I believe we need to conform to `HealthKitConstraint` to use SpeziHealthKit in the application?

Reply:
We can remove ObservableObject, but I believe the Standard [needs to conform](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/0.5.3/documentation/spezihealthkit#Example) to HealthKitConstraint to use SpeziHealthKit in the application, otherwise it doesn't build, even if the Standard itself is not doing anything with the HealthKit data.

Reply:
On a different topic, we are only really using SpeziHealthKit here for onboarding currently, whereas `HealthDataFetcher` is actually performing the HealthKit queries. This could probably be consolidated.